proto/quicksilver/interchainstaking/v1/proposals.proto

```diff
@@ -32,59 +30,51 @@ message RegisterZoneProposalWithDeposit {
   option (gogoproto.equal) = false;
   option (gogoproto.goproto_getters) = false;
 
-  string title = 1 [ (gogoproto.moretags) = "yaml:\"title\"" ];
-  string description = 2 [ (gogoproto.moretags) = "yaml:\"description\"" ];
-
-  string connection_id = 3 [ (gogoproto.moretags) = "yaml:\"connection_id\"" ];
-  string base_denom = 4 [ (gogoproto.moretags) = "yaml:\"base_denom\"" ];
-  string local_denom = 5 [ (gogoproto.moretags) = "yaml:\"local_denom\"" ];
-  string account_prefix = 6
-      [ (gogoproto.moretags) = "yaml:\"account_prefix\"" ];
-  bool multi_send = 7 [ (gogoproto.moretags) = "yaml:\"multi_send\"" ];
-  bool liquidity_module = 8
-      [ (gogoproto.moretags) = "yaml:\"liquidity_module\"" ];
-  string deposit = 9 [ (gogoproto.moretags) = "yaml:\"deposit\"" ];
-  bool return_to_sender = 10
-      [ (gogoproto.moretags) = "yaml:\"return_to_sender\"" ];
-  bool deposits_enabled = 11
-      [ (gogoproto.moretags) = "yaml:\"deposits_enabled\"" ];
-  bool unbonding_enabled = 12
-      [ (gogoproto.moretags) = "yaml:\"deposits_enabled\"" ];
-  int64 decimals = 13 [ (gogoproto.moretags) = "yaml:\"decimals\"" ];
+  string title = 1 [(gogoproto.moretags) = "yaml:\"title\""];
+  string description = 2 [(gogoproto.moretags) = "yaml:\"description\""];
+
+  string connection_id = 3 [(gogoproto.moretags) = "yaml:\"connection_id\""];
+  string base_denom = 4 [(gogoproto.moretags) = "yaml:\"base_denom\""];
+  string local_denom = 5 [(gogoproto.moretags) = "yaml:\"local_denom\""];
+  string account_prefix = 6 [(gogoproto.moretags) = "yaml:\"account_prefix\""];
+  bool multi_send = 7 [(gogoproto.moretags) = "yaml:\"multi_send\""];
+  bool liquidity_module = 8 [(gogoproto.moretags) = "yaml:\"liquidity_module\""];
+  string deposit = 9 [(gogoproto.moretags) = "yaml:\"deposit\""];
+  bool return_to_sender = 10 [(gogoproto.moretags) = "yaml:\"return_to_sender\""];
+  bool deposits_enabled = 11 [(gogoproto.moretags) = "yaml:\"deposits_enabled\""];
+  bool unbonding_enabled = 12 [(gogoproto.moretags) = "yaml:\"deposits_enabled\""];
```

Comment:
```suggestion
  bool unbonding_enabled = 12 [(gogoproto.moretags) = "yaml:\"unbonding_enabled\""];
```

hasn't caused us issues before but lets fix this now before it bites us in the ass later!